src/pages/user.tsx

```diff
@@ -35,6 +35,10 @@ export function User() {
     setShowModal(true);
   };
 
+  const closeModal = () => {
```

Comment:
Could you please change the name to a more meaningful one?

Reply:
I modified the setShowModal function to setModalVisible, which provides a clearer indication of its role. However, I retained the word 'modal' in the name because I intend to use this action for other modals as well. For instance, it will be utilized for a modal to create a new user or edit existing user information.